aplib.net-demo/Assets/Testing/AplibTests/MyFirstAplibTest.cs

```diff
@@ -13,8 +13,9 @@
 
 namespace Testing.AplibTests
 {
-    public class MyFirstBeliefSet : BeliefSet
+    public class MyFirstAplibTestBeliefSet : BeliefSet
```

Comment:
Sorry for the confusion, but I just meant please rename it to what it was, not MyFirstAplibTestBeliefSet but MyFirstBeliefSet